packages/by-name/kata/kata-kernel-uvm/package.nix

```diff
@@ -22,7 +22,8 @@ let
     postPatch = ''
       config=$(find . -regex '.*/config-[0-9.-]+-confidential')
       substituteInPlace $config \
-        --replace-fail 'CONFIG_INITRAMFS_SOURCE="initramfs.cpio.gz"' 'CONFIG_INITRAMFS_SOURCE=""'
+        --replace-fail 'CONFIG_INITRAMFS_SOURCE="initramfs.cpio.gz"' 'CONFIG_INITRAMFS_SOURCE=""' \
+        --replace-fail '# CONFIG_DM_INIT is not set' 'CONFIG_DM_INIT=y'
```

Comment:
We should probably add a little comment here explaining why (both of) the patches are necessary.
For the first one, we should mention that we're not using an initrd.